### ISSUE TITLE
Decorators in link options not works

### ISSUE THREAD

**Issue by oriolfg** [numaxlab/nova-ckeditor5-classic#18]
Adding decorators to the link options, in the settings, no change is observed, it makes me think it is not working properly.

For example:

```php
'options' => [
        'language' => 'en',
        'toolbar' => [
            'Link',
        ],
        'link' => [
            'decorators' => [
                'isExternal' => [
                    'mode' => 'manual',
                    'label' => 'Open in a new tab?',
                    'attributes' => [
                        'target' => '_blank',
                        'rel'=> 'noopener noreferrer'
                    ]
                ],
                'noFollow' => [
                    'mode' => 'manual',
                    'label' => 'No Follow',
                    'attributes' => [
                        'rel' => 'nofollow'
                    ]
                ]
            ]
        ],
    ]
```

Link functionality appears normal and "NoFollow" or "Open in a new tab" options are not displayed

Do I need to do any extra installation?

(I use the latest version of your package, on an installation with laravel 7 and new 3)

**Comment by oriolfg** [comment]
I managed to make it work, redoing the compiled webpack file "dist / js / field.js. 

When redoing this file with the configuration file with the link plugin then it works.

Steps:
1 -Fork repo or create local copy and use it instead numaxlab repository
2 - Add link options to default configurations file
3 - Create new field.js:
```bash
npm i
npm run dev
rm -rf node_modules
```
4 - Enjoy your ckeditor with link options working fine! But forgot about the original repository updates :(

**Comment by oriolfg** [comment]
You will probably need to resolve this in your repository for it to work properly

**Comment by ablunier** [comment]
Hi @oriolfg, thanks for your feedback. This wasn't working because the functionality was not included in the latest release.
Now it is included in [v1.1.2](https://github.com/numaxlab/nova-ckeditor5-classic/releases/tag/1.1.2)
